src/fidesops/schemas/saas/saas_config.py

```diff
@@ -159,6 +160,36 @@ def validate_grouped_inputs(cls, values: Dict[str, Any]) -> Dict[str, Any]:
 
         return values
 
+    @root_validator
+    def validate_override(cls, values: Dict[str, Any]) -> Dict[str, Any]:
+        """Validate that configs related to request overrides are set properly"""
+        if not values.get("request_override"):
+            if not values.get("path"):
+                raise ValueError(
+                    "A request must specify a path if no request_override is provided"
+                )
+            if not values.get("method"):
+                raise ValueError(
+                    "A request must specify a method if no request_override is provided"
+                )
+
+        else:  # if a request override is specified, many fields are NOT allowed
+            invalid = next(
+                (
+                    k
+                    for k in values.keys()
+                    if values.get(k)
+                    and k not in ("request_override", "param_values", "grouped_inputs")
+                ),
+                None,
+            )
+            if invalid:
+                raise ValueError(
+                    f"Invalid property {invalid} on a request with request_override specified"
```

Comment:
If I'm reading this correctly, we'll only return the first invalid field. I think it would be better if we returned a list of all the invalid fields at once, so the user can fix them all at once instead of removing one then getting a different error.